dpbench/benchmarks/l2_norm/l2_norm_sycl_native_ext/l2_norm_sycl/_l2_norm_sycl.cpp

```diff
@@ -0,0 +1,46 @@
+//==- _l2_norm_sycl.cpp - Python native extension of l2-norm   ===//
+//
+// Copyright 2022 Intel Corp.
+//
+// SPDX - License - Identifier : Apache 2.0
+///
+/// \file
+/// The files implements a SYCL-based Python native extension for the
+/// l2-norm benchmark.
+
+#include "_l2_norm_kernel.hpp"
+#include <CL/sycl.hpp>
+#include <dpctl4pybind11.hpp>
+#include <iostream>
+#include <stdlib.h>
+#include <type_traits>
+#include <vector>
+
+using namespace sycl;
+namespace py = pybind11;
+
+void l2_norm_sync(dpctl::tensor::usm_ndarray a, dpctl::tensor::usm_ndarray d)
+{
+    // sycl::event res_ev;
```

Comment:
Please remove commented out line of code.

Reply:
Cleaned.